libp2p/NodeTable.cpp

```diff
@@ -482,6 +490,20 @@ void NodeTable::onPacketReceived(
 
             case FindNode::type:
             {
+                auto const sourceNodeEntry = nodeEntry(packet->sourceid);
+                if (!sourceNodeEntry)
+                {
+                    LOG(m_logger) << "Source node (" << packet->sourceid << "@" << _from
+                                  << ") not found in node table. Ignoring FindNode request.";
+                    return;
+                }
+                if (sourceNodeEntry->lastPongReceivedTime + c_bondingTimeSeconds <
```

Comment:
Maybe a helper for this check would be nice, because now it's repeated in two places.

Reply:
Making this a helper on NodeEntry